src/MSBuild/Resources/Strings.resx

```diff
@@ -633,7 +633,7 @@
                      instead of multiple processes.
                      This option is not supported.
                      The maximum number of threads used is determined by the
-                     -maxCpuCount switch.
+                     -maxCpuCount switch. (recommended not greater than 256)
```

Comment:
it's not only recommended, it's required as of the code's current state. I'd prefer if you resolved it as Alina recommended: cut it down to 256 instead of error

Reply:
Agree. Done